lib/commands/start.js

```diff
@@ -32,16 +32,15 @@ class StartCommand extends Command {
             return;
         }
 
-        const url = instance.config.get('url');
+        instance.checkEnvironment();
 
-        if (this.system.environment === 'production' && url.startsWith('http://')) {
+        if (this.system.environment === 'production' && instance.config.get('url', '').startsWith('http://')) {
```

Comment:
Related question - should Ghost start if there isn't a URL? 😄

Reply:
probably not - though I feel like that's probably a check for a different place in the code 😅